### ISSUE TITLE
Twitter cards not functional?

### ISSUE THREAD

**Issue by jacobwilliams** [pelican-plugins/seo#51]
I set everything to True

```python
SEO_REPORT = True
SEO_ENHANCER = True
SEO_ENHANCER_OPEN_GRAPH = True
SEO_ENHANCER_TWITTER_CARDS = True
```

But I'm not seeing any `twitter:card` tags in the html. Using Python 3.8.5 on Mac. I can work up a simple example if necessary (are there examples somewhere already?)

**Comment by kevingoldsmith** [comment]
this didn't work for me.

For me, no matter what is in the config, I always get the report, I always get Open Graph, but never twitter cards.

I tried:
```
import pelican
from pelican.plugins import sitemap, seo

installed_plugins_dir = pelican.plugins.__path__[0]   # ones that are pip installed
local_plugins_dir = 'plugins'  # ones that are in the repo

PLUGIN_PATHS = [local_plugins_dir, installed_plugins_dir]
PLUGINS = ['seo', 'sitemap']

SEO_REPORT = False
SEO_ENHANCER = True
SEO_ENHANCER_OPEN_GRAPH = True
SEO_ENHANCER_TWITTER_CARDS = True
LOGO = "https://nimbleautonomy.com/android-chrome-512x512.png"

```

Has the Exact same result as (none of the paths or imports or plugins list stuff from above):
```
SEO_REPORT = False
SEO_ENHANCER = True
SEO_ENHANCER_OPEN_GRAPH = True
SEO_ENHANCER_TWITTER_CARDS = True
LOGO = "https://nimbleautonomy.com/android-chrome-512x512.png"

```

The confusing thing is that the OG stuff is not on by default, so something weird is going on.

**Is the only way to get twitter cards to install to the local plugins folder?**

**Comment by EpocDotFr** [comment]
Exact same issue here.

**Comment by ianrussel** [comment]
> Exact same issue here.
> 
> Edit: fixes mentioned above have not been released already, you have to install the package from the latest git commit: `pip install git+https://github.com/pelican-plugins/seo.git@5a1810df158532ffc3ea8b84a0e88b26d9594f70#egg=pelican-seo`

Thus this works for you?

**Comment by EpocDotFr** [comment]
Of course.

**Comment by justinmayer** [comment]
I just released [SEO 1.3](https://github.com/pelican-plugins/seo/releases/tag/1.3.0), which contains changes that should resolve this issue.
